Binary files 2.9.2_orig/.DS_Store and 2.9.2/.DS_Store differ
diff -ruN 2.9.2_orig/make.files/simset.make 2.9.2/make.files/simset.make
--- 2.9.2_orig/make.files/simset.make	2013-07-30 20:19:26.000000000 +0100
+++ 2.9.2/make.files/simset.make	2019-06-03 10:53:26.164586631 +0100
@@ -59,7 +59,7 @@
 PHG_SRC_USR = $(SIMSET_PATH_USR)/src
 OBJ_DIR = $(SIMSET_PATH_USR)/obj
 PROGRAMS = $(SIMSET_PATH_USR)/bin
-
+LIB_DIR = $(SIMSET_PATH_USR)/lib
 
 # Set OS Flags; all Unix systems should set GEN_UNIX.
 # Choose one of the following for CFLAGS:
@@ -85,11 +85,11 @@
 # WINNT
 
 # Suggested compiler flags for Macintosh Unix (Darwin)
-OS_CFLAGS = -DDARWIN -g
+#OS_CFLAGS = -DDARWIN -g
 
 # LINUX
 # Suggested compiler flags for LINUX
-# OS_CFLAGS = -DGEN_UNIX -DLINUX
+OS_CFLAGS = -DGEN_UNIX -DLINUX -Wall -fPIC
 
 
 COMPILER = cc
@@ -97,14 +97,15 @@
 # Select the debug or nodebug CFLAGS option:  debug has added data checking
 #  and is recommended
 # Choose between the last two options instead if your compiler does not yet support -iquote
-CFLAGS = ${OS_CFLAGS} -c -DPHG_DEBUG -iquote"${PHG_SRC}" -DkSimSET_Path='"${SIMSET_PATH}"' 
+# CFLAGS = ${OS_CFLAGS} -c -DPHG_DEBUG -iquote"${PHG_SRC}" -DkSimSET_Path='"${SIMSET_PATH}"' 
 # CFLAGS = ${OS_CFLAGS} -c -iquote"${PHG_SRC}" -DkSimSET_Path='"${SIMSET_PATH}"' 
 # 
-# CFLAGS = ${OS_CFLAGS} -c -DPHG_DEBUG -I"${PHG_SRC}" -DkSimSET_Path='"${SIMSET_PATH}"' 
+CFLAGS = ${OS_CFLAGS} -c -DPHG_DEBUG -I"${PHG_SRC}" -DkSimSET_Path='"${SIMSET_PATH}"' 
 # CFLAGS = ${OS_CFLAGS} -c -I"${PHG_SRC}" -DkSimSET_Path='"${SIMSET_PATH}"' 
 
 
 PROGRAM = ${PROGRAMS}/simset
+LIBRARY = ${LIB_DIR}/libsimset.so
 MKFILE = ${MKFILES}/simset.make
 
 
@@ -230,11 +231,16 @@
 
 # Linking instructions
 
-${PROGRAM}:: ${MKFILE} $(OBJECTS)
-	echo linking
-	${COMPILER}  -o ${PROGRAM} $(OBJECTS)   -lm 
+all : ${LIBRARY} ${PROGRAM}
 
+${LIBRARY}: ${MKFILE} $(OBJECTS)
+	echo linking
+	${COMPILER} -shared -Wl,-soname,${LIBRARY} -o ${LIBRARY} $(OBJECTS)
 
+${PROGRAM}: ${MKFILE} $(OBJECTS)
+	echo linking
+	${COMPILER}  -o ${PROGRAM} $(OBJECTS)   -lm 
+	
 
 # Compiling instructions

diff -ruN 2.9.2_orig/src/bin.phg.c 2.9.2/src/bin.phg.c
--- 2.9.2_orig/src/bin.phg.c	2013-07-15 10:53:34.000000000 +0100
+++ 2.9.2/src/bin.phg.c	2019-03-30 16:57:08.522896562 +0000
@@ -96,85 +96,13 @@
  *********************************************************************************/
 #define PHG_BIN_MAIN	/* Note we are substituting ourselves for phg's main */
 
-#include <stdio.h>
-#include <string.h>
-
-#include "SystemDependent.h"
-
-#include "LbTypes.h"
-#include "LbError.h"
-#include "LbDebug.h"
-#include "LbEnvironment.h"
-#include "LbFile.h"
-#include "LbMemory.h"
-#include "LbParamFile.h"
-#include "LbInterface.h"
-#include "LbHeader.h"
-
-#include "Photon.h"
-#include "PhgParams.h"
-#include "ColTypes.h"
-#include "ColParams.h"
-#include "DetTypes.h"
-#include "DetParams.h"
-#include "CylPos.h"
-#include "PhgMath.h"
-#include "PhoHFile.h"
-#include "PhgHdr.h"
-#include "ProdTbl.h"
-#include "PhoTrk.h"
-#include "SubObj.h"
-#include "EmisList.h"
-#include "Collimator.h"
-#include "Detector.h"
-#include "phg.h"
-#include "PhgBin.h"
+#include "bin.phg.h"
 
 
 #ifdef MPW
 #pragma segment PHG_BIN_MAIN
 #endif
 
-/* LOCAL CONSTANTS */
-#define PHGRDHST_IsUsePHGHistory()		LbFgIsSet(PhgOptions, LBFlag0)		/* Will we use the PHG history file */
-#define PHGRDHST_IsUseColHistory()		LbFgIsSet(PhgOptions, LBFlag1)		/* Will we use the Collimator history file */
-#define PHGRDHST_IsUseDetHistory()		LbFgIsSet(PhgOptions, LBFlag2)		/* Will we use the Detector history file */
-
-#define	PHGRDHST_NumFlags	3													/* Number of flags defined */
-
-/* LOCAL TYPES */
-typedef enum  {Null, Decay, Photon} EventTy;
-
-/* LOCAL GLOBALS */
-static Boolean				phgrdhstCanceled;				/* Global cancelation flag */
-static CollimatedPhotonsTy	phgrdhstColPhotons;				/* These are the successfully collimated photons */
-static DetectedPhotonsTy	phgrdhstDetPhotons;				/* These are the successfully detected photons */
-static char					phgrdhstErrStr[1024];			/* Error string storage */
-static LbUsFourByte			phgrdhstNumToProc;				/* Number of histories to process */
-static char					phgrdhstHistName[1024];			/* Name of history file */
-static char					phgrdhstHistParamsName[1024];	/* Name of history parameters file */
-static LbUsFourByte			phgrdhstArgIndex;
-static ProdTblProdTblInfoTy	phgrdhstPrdTblInfo;				/* Info for initializing productivity table */
-static PhoHFileHdrTy		phgrdhstHdrParams;				/* Input header */
-
-/* PROTOTYPES */
-Boolean 		phgrdhstInitialize(int argc, char *argv[]);
-void			phgrdhstTerminate(void);
-Boolean			phgbin(int argc, char *argv[]);
-EventTy			readEvent(FILE *historyFile,
-						  PHG_Decay *decayPtr,
-						  PHG_DetectedPhoton *photonPtr);
-EventTy			oldReadEvent(FILE *historyFile,
-                             PHG_Decay *decayPtr,
-                             PHG_DetectedPhoton *photonPtr,
-                             Boolean isOldPhotons1,
-                             Boolean isOldPhotons2);
-Boolean			phgrdhstStandard(char *argv[]);
-Boolean			phgrdhstCustom(char *argv[]);
-void			phgbinProcessPhotons(PhoHFileHkTy *histHk, PHG_Decay *decayPtr,
-                                     PHG_TrackingPhoton *bluePhotons, LbUsFourByte numBlues,
-                                     PHG_TrackingPhoton *pinkPhotons, LbUsFourByte numPinks);
-
 
 /* FUNCTIONS */
 /**********************
@@ -220,7 +148,7 @@
 		if ((phgrdhstArgIndex != 0) && (argv[phgrdhstArgIndex] != 0)) {
             
 			/* Get first param file and save number of param files to process */
-			strcpy(PhgRunTimeParams.PhgParamFilePath,argv[phgrdhstArgIndex]);
+            strcpy(PhgRunTimeParams.PhgParamFilePath,argv[phgrdhstArgIndex]);
 			phgrdhstNumToProc = (argc - phgrdhstArgIndex);
 		}
 		else {
@@ -682,7 +610,8 @@
 						eventType = readEvent(historyFile, &nextDecay, &detectedPhoton);
 					}
 					
-					while ( eventType == Photon) {
+                    while ( eventType == Photon)
+                    {
                         
 						/* Convert to a tracking photon */
 						{
diff -ruN 2.9.2_orig/src/bin.phg.h 2.9.2/src/bin.phg.h
--- 2.9.2_orig/src/bin.phg.h	1970-01-01 01:00:00.000000000 +0100
+++ 2.9.2/src/bin.phg.h	2019-03-28 16:48:00.601727849 +0000
@@ -0,0 +1,80 @@
+#ifndef BIN_PHG_HDR
+#define BIN_PHG_HDR
+
+#include <stdio.h>
+#include <string.h>
+
+#include "SystemDependent.h"
+
+#include "LbTypes.h"
+#include "LbError.h"
+#include "LbDebug.h"
+#include "LbEnvironment.h"
+#include "LbFile.h"
+#include "LbMemory.h"
+#include "LbParamFile.h"
+#include "LbInterface.h"
+#include "LbHeader.h"
+
+#include "Photon.h"
+#include "PhgParams.h"
+#include "ColTypes.h"
+#include "ColParams.h"
+#include "DetTypes.h"
+#include "DetParams.h"
+#include "CylPos.h"
+#include "PhgMath.h"
+#include "PhoHFile.h"
+#include "PhgHdr.h"
+#include "ProdTbl.h"
+#include "PhoTrk.h"
+#include "SubObj.h"
+#include "EmisList.h"
+#include "Collimator.h"
+#include "Detector.h"
+#include "phg.h"
+#include "PhgBin.h"
+
+/* LOCAL CONSTANTS */
+#define PHGRDHST_IsUsePHGHistory()		LbFgIsSet(PhgOptions, LBFlag0)		/* Will we use the PHG history file */
+#define PHGRDHST_IsUseColHistory()		LbFgIsSet(PhgOptions, LBFlag1)		/* Will we use the Collimator history file */
+#define PHGRDHST_IsUseDetHistory()		LbFgIsSet(PhgOptions, LBFlag2)		/* Will we use the Detector history file */
+
+#define	PHGRDHST_NumFlags	3													/* Number of flags defined */
+
+/* LOCAL TYPES */
+typedef enum  {Null, Decay, Photon} EventTy;
+
+/* LOCAL GLOBALS */
+static Boolean				phgrdhstCanceled;				/* Global cancelation flag */
+static CollimatedPhotonsTy	phgrdhstColPhotons;				/* These are the successfully collimated photons */
+static DetectedPhotonsTy	phgrdhstDetPhotons;				/* These are the successfully detected photons */
+static char					phgrdhstErrStr[1024];			/* Error string storage */
+static LbUsFourByte			phgrdhstNumToProc;				/* Number of histories to process */
+static char					phgrdhstHistName[1024];			/* Name of history file */
+static char					phgrdhstHistParamsName[1024];	/* Name of history parameters file */
+static LbUsFourByte			phgrdhstArgIndex;
+static ProdTblProdTblInfoTy	phgrdhstPrdTblInfo;				/* Info for initializing productivity table */
+static PhoHFileHdrTy		phgrdhstHdrParams;				/* Input header */
+
+/* PROTOTYPES */
+Boolean 		phgrdhstInitialize(int argc, char *argv[]);
+void			phgrdhstTerminate(void);
+Boolean			phgbin(int argc, char *argv[]);
+EventTy			readEvent(FILE *historyFile,
+                                                  PHG_Decay *decayPtr,
+                                                  PHG_DetectedPhoton *photonPtr);
+EventTy			oldReadEvent(FILE *historyFile,
+                             PHG_Decay *decayPtr,
+                             PHG_DetectedPhoton *photonPtr,
+                             Boolean isOldPhotons1,
+                             Boolean isOldPhotons2);
+Boolean			phgrdhstStandard(char *argv[]);
+Boolean			phgrdhstCustom(char *argv[]);
+void			phgbinProcessPhotons(PhoHFileHkTy *histHk, PHG_Decay *decayPtr,
+                                     PHG_TrackingPhoton *bluePhotons, LbUsFourByte numBlues,
+                                     PHG_TrackingPhoton *pinkPhotons, LbUsFourByte numPinks);
+
+
+
+#endif
diff -ruN 2.9.2_orig/src/combine.hist.c 2.9.2/src/combine.hist.c
--- 2.9.2_orig/src/combine.hist.c	2013-07-15 10:53:34.000000000 +0100
+++ 2.9.2/src/combine.hist.c	2019-06-03 10:40:47.105344114 +0100
@@ -103,7 +103,7 @@
 	char				*dataBuffPtr;				/* Buffer for reading in data */
 	FILE				*inputFile;					/* Current input file */
 	FILE				*outputFile;				/* The output file */
-	LbUsFourByte		hdrSize;					/* Size of the file's header */
+    LbUsFourByte		hdrSize = 32768;					/* Size of the file's header */
 	LbUsFourByte		curFileIndex;				/* Current file index */
 	LbUsFourByte		numToProcess;				/* Number of files to process */
 	LbUsFourByte		numRead;					/* Number of bytes read */
@@ -159,21 +159,21 @@
 		}
 		
 		/* Open first input file */
-		if ((inputFile = LbFlFileOpen(argv[curFileIndex], "rb")) == 0) {
-			sprintf(errStr, "Unable to open input file\n'%s'.", argv[curFileIndex]);
-			ErStFileError(errStr);
-			break;
-		}
+        if ((inputFile = LbFlFileOpen(argv[curFileIndex], "rb")) == 0) {
+            sprintf(errStr, "Unable to open input file\n'%s'.", argv[curFileIndex]);
+            ErStFileError(errStr);
+            break;
+        }
 
 		/* Read the header in the first input file */
-		{
+        {
 			/* First, read the header size; it is in the first four bytes */
-			if (fread(&hdrSize, sizeof(LbUsFourByte), 1, inputFile) != 1) {
-				sprintf(errStr, "\nUnable to read header size from input file '%s'.\n",
-					argv[curFileIndex]);
-				ErStFileError(errStr);
-				break;
-			}
+//            if (fread(&hdrSize, sizeof(LbUsFourByte), 1, inputFile) != 1) {
+//                sprintf(errStr, "\nUnable to read header size from input file '%s'.\n",
+//                    argv[curFileIndex]);
+//                ErStFileError(errStr);
+//                break;
+//            }
 			
 			/* Verify we know how to deal with this */
 			if (hdrSize == sizeof(header)) {
@@ -191,21 +191,21 @@
 			}
 			
 			/* Reset to zero and read in the header */
-			if (fseek(inputFile, 0, SEEK_SET) != 0) {
-				ErStFileError("\nUnable to reset to beginning of history file.");
-				break;
-			}
+            if (fseek(inputFile, 0, SEEK_SET) != 0) {
+                ErStFileError("\nUnable to reset to beginning of history file.");
+                break;
+            }
 
 			/* Read in the header */
-			if (fread(hdrBuffPtr, hdrSize, 1, inputFile) != 1) {
-				ErStFileError("\nUnable to read header from history file.");
-				break;
-			}
+            if (fread(hdrBuffPtr, hdrSize, 1, inputFile) != 1) {
+                ErStFileError("\nUnable to read header from history file.");
+                break;
+            }
 
 			/* Save the header */
 			if (updateHdr == true){
 				memcpy(&newHeader, &header, sizeof(header));
-			}
+            }
 
 		}
 
@@ -268,10 +268,10 @@
 					break;
 				}
 				/* First, read the header size; it is in the first four bytes */
-				if (fread(&hdrSize, sizeof(LbUsFourByte), 1, inputFile) != 1) {
-					ErStFileError("\nUnable to read header size from input file.");
-					break;
-				}
+//                if (fread(&hdrSize, sizeof(LbUsFourByte), 1, inputFile) != 1) {
+//					ErStFileError("\nUnable to read header size from input file.");
+//					break;
+//                }
 				
 				/* Allocate header buffer if unknown */
 				if (updateHdr == false) {
diff -ruN 2.9.2_orig/src/LbTypes.h 2.9.2/src/LbTypes.h
--- 2.9.2_orig/src/LbTypes.h	2014-01-24 16:26:37.000000000 +0000
+++ 2.9.2/src/LbTypes.h	2019-03-16 16:27:11.119406411 +0000
@@ -77,7 +77,7 @@
 
 /* Define this value if your system has the file /usr/include/sys/types.h
     (otherwise comment out the following line).  */
-/* #define LB_TYPE_USE_SYS_INTS */
+#define LB_TYPE_USE_SYS_INTS
 
 
 #include <float.h>
diff -ruN 2.9.2_orig/src/PhgParams.c 2.9.2/src/PhgParams.c
--- 2.9.2_orig/src/PhgParams.c	2013-07-25 02:36:01.000000000 +0100
+++ 2.9.2/src/PhgParams.c	2019-03-26 22:44:55.991673505 +0000
@@ -97,8 +97,6 @@
 /* LOCAL GLOBALS */
 static 	char		phgBinErrString[512];				/* Storage for error string s*/
 
-static	LbPfHkTy	phgParamFileHk;				/* Our parameter file */
-
 /* When changing the following list also change PhgEn_RunTimeParamsTy in PhgParams.h.
 The two lists must have the same order */
 static	labelTy		phgRunTimeParamLabels[] = {	/* The parameter labels */
diff -ruN 2.9.2_orig/src/PhgParams.h 2.9.2/src/PhgParams.h
--- 2.9.2_orig/src/PhgParams.h	2013-07-15 10:53:35.000000000 +0100
+++ 2.9.2/src/PhgParams.h	2019-03-26 22:45:11.478506154 +0000
@@ -606,6 +606,8 @@
 *********************************************************************************/
 #define PHGGetLengthOfScan() PhgRunTimeParams.Phg_LengthOfScan
 
+static	LbPfHkTy	phgParamFileHk;				/* Our parameter file */
+
 /* PROTOTYPES */
 PhgEn_RunTimeParamsTy	PhgLookupRunTimeParamLabel(char *label);
 Boolean					PhgGetRunTimeParams(void);
diff -ruN 2.9.2_orig/src/print.header.c 2.9.2/src/print.header.c
--- 2.9.2_orig/src/print.header.c	2013-07-25 02:35:00.000000000 +0100
+++ 2.9.2/src/print.header.c	2019-03-17 14:13:12.877848060 +0000
@@ -50,51 +50,9 @@
 *
 *********************************************************************************/
 
-#include <stdio.h>
-#include <string.h>
+#include "print.header.h"
 
-#include "SystemDependent.h"
 
-#include "LbTypes.h"
-#include "LbError.h"
-#include "LbDebug.h"
-#include "LbEnvironment.h"
-#include "LbFile.h"
-#include "LbMemory.h"
-#include "LbInterface.h"
-#include "LbParamFile.h"
-#include "LbHeader.h"
-
-#include "Photon.h"
-#include "PhgParams.h"
-#include "ColTypes.h"
-#include "ColParams.h"
-#include "DetTypes.h"
-#include "DetParams.h"
-#include "CylPos.h"
-#include "PhgMath.h"
-#include "PhoHFile.h"
-#include "ProdTbl.h"
-#include "PhoTrk.h"
-#include "SubObj.h"
-#include "EmisList.h"
-#include "Collimator.h"
-#include "Detector.h"
-#include "PhgHdr.h"
-#include "phg.h"
-#include "PhgBin.h"
-
-/* LOCAL CONSTANTS */
-
-/* LOCAL TYPES */
-
-/* LOCAL GLOBALS */
-static	PhoHFileHdrTy		header;					/* The history file header */
-static	LbHdrHkTy			headerHk;				/* The header hook */
-
-/* PROTOTYPES */
-Boolean			PrintHeader(int argc, char *argv[]);
-static	void			display(PhoHFileHdrTy *headerPtr);
 
 /* FUNCTIONS */
 
diff -ruN 2.9.2_orig/src/print.header.h 2.9.2/src/print.header.h
--- 2.9.2_orig/src/print.header.h	1970-01-01 01:00:00.000000000 +0100
+++ 2.9.2/src/print.header.h	2019-03-17 14:13:10.067802670 +0000
@@ -0,0 +1,52 @@
+#ifndef PRINT_HEADER
+#define PRINT_HEADER
+
+
+#include <stdio.h>
+#include <string.h>
+
+#include "SystemDependent.h"
+
+#include "LbTypes.h"
+#include "LbError.h"
+#include "LbDebug.h"
+#include "LbEnvironment.h"
+#include "LbFile.h"
+#include "LbMemory.h"
+#include "LbInterface.h"
+#include "LbParamFile.h"
+#include "LbHeader.h"
+
+#include "Photon.h"
+#include "PhgParams.h"
+#include "ColTypes.h"
+#include "ColParams.h"
+#include "DetTypes.h"
+#include "DetParams.h"
+#include "CylPos.h"
+#include "PhgMath.h"
+#include "PhoHFile.h"
+#include "ProdTbl.h"
+#include "PhoTrk.h"
+#include "SubObj.h"
+#include "EmisList.h"
+#include "Collimator.h"
+#include "Detector.h"
+#include "PhgHdr.h"
+#include "phg.h"
+#include "PhgBin.h" 
+
+
+/* LOCAL CONSTANTS */
+
+/* LOCAL TYPES */
+
+/* LOCAL GLOBALS */
+static	PhoHFileHdrTy		header;					/* The history file header */
+static	LbHdrHkTy			headerHk;				/* The header hook */
+
+/* PROTOTYPES */
+Boolean			PrintHeader(int argc, char *argv[]);
+static	void			display(PhoHFileHdrTy *headerPtr);
+
+#endif
diff -ruN 2.9.2_orig/src/SimSET.cflags 2.9.2/src/SimSET.cflags
--- 2.9.2_orig/src/SimSET.cflags	1970-01-01 01:00:00.000000000 +0100
+++ 2.9.2/src/SimSET.cflags	2019-06-02 23:53:31.541319043 +0100
@@ -0,0 +1 @@
+-std=c17
\ No newline at end of file
diff -ruN 2.9.2_orig/src/SimSET.config 2.9.2/src/SimSET.config
--- 2.9.2_orig/src/SimSET.config	1970-01-01 01:00:00.000000000 +0100
+++ 2.9.2/src/SimSET.config	2019-04-03 13:04:47.740633671 +0100
@@ -0,0 +1,2 @@
+// Add predefined macros for your project here. For example:
+// #define THE_ANSWER 42
diff -ruN 2.9.2_orig/src/SimSET.creator 2.9.2/src/SimSET.creator
--- 2.9.2_orig/src/SimSET.creator	1970-01-01 01:00:00.000000000 +0100
+++ 2.9.2/src/SimSET.creator	2019-04-03 13:04:47.740633671 +0100
@@ -0,0 +1 @@
+[General]
diff -ruN 2.9.2_orig/src/SimSET.cxxflags 2.9.2/src/SimSET.cxxflags
--- 2.9.2_orig/src/SimSET.cxxflags	1970-01-01 01:00:00.000000000 +0100
+++ 2.9.2/src/SimSET.cxxflags	2019-06-02 23:53:31.531319263 +0100
@@ -0,0 +1 @@
+-std=c++17
\ No newline at end of file
diff -ruN 2.9.2_orig/src/SimSET.files 2.9.2/src/SimSET.files
--- 2.9.2_orig/src/SimSET.files	1970-01-01 01:00:00.000000000 +0100
+++ 2.9.2/src/SimSET.files	2019-04-03 13:04:47.733966960 +0100
@@ -0,0 +1,122 @@
+ColParams.c
+ColParams.h
+ColSlat.c
+ColSlat.h
+ColTypes.h
+ColUsr.c
+ColUsr.h
+Collimator.c
+Collimator.h
+CylPos.c
+CylPos.h
+DetBlock.c
+DetBlock.h
+DetCylinder.c
+DetCylinder.h
+DetGeometric.c
+DetGeometric.h
+DetParams.c
+DetParams.h
+DetPlanar.c
+DetPlanar.h
+DetTypes.h
+DetUsr.c
+DetUsr.h
+Detector.c
+Detector.h
+EmisList.c
+EmisList.h
+Lb2DGeometry.c
+Lb2DGeometry.h
+LbConvert.c
+LbConvert.h
+LbDebug.c
+LbDebug.h
+LbEnvironment.c
+LbEnvironment.h
+LbError.c
+LbError.h
+LbFile.c
+LbFile.h
+LbHeader.c
+LbHeader.h
+LbInterface.c
+LbInterface.h
+LbMacros.h
+LbMath.h
+LbMemory.c
+LbMemory.h
+LbParamFile.c
+LbParamFile.h
+LbSort.c
+LbSort.h
+LbTiming.c
+LbTiming.h
+LbTypes.h
+MT19937.c
+MT19937.h
+PhgBin.c
+PhgBin.h
+PhgHdr.c
+PhgHdr.h
+PhgIsotopes.c
+PhgIsotopes.h
+PhgMath.c
+PhgMath.h
+PhgParams.c
+PhgParams.h
+PhgUsrBin.c
+PhgUsrBin.h
+PhoHFile.c
+PhoHFile.h
+PhoHStat.c
+PhoHStat.h
+PhoTrk.c
+PhoTrk.h
+Photon.h
+ProdTbl.c
+ProdTbl.h
+SubObj.c
+SubObj.h
+SystemDependent.h
+UNCCollimator.c
+UNCCollimator.h
+addrandUsr.c
+addrandUsr.h
+addrandoms.c
+addrandoms.h
+atten.correct.c
+bcomp.c
+bin.phg.c
+bin.phg.h
+breakpoint.swap.c
+build_att.c
+build_coh.c
+calc.attenuation.c
+calc.attenuation.h
+collapse.c
+collapse3d.c
+combine.bin.c
+combine.hist.c
+convert.c
+convert.coh.c
+convert.header.c
+display.header.c
+extract.c
+extract.lines.c
+line3d.c
+makeindexfile.c
+migrate.c
+phg.c
+phg.h
+phg.swap.c
+print.header.c
+print.header.h
+reorder.c
+resampledecaytime.c
+reverse.bytes.c
+scale.c
+simset.c
+strip.header.c
+timesort.c
+ttest.c
diff -ruN 2.9.2_orig/src/SimSET.includes 2.9.2/src/SimSET.includes
--- 2.9.2_orig/src/SimSET.includes	1970-01-01 01:00:00.000000000 +0100
+++ 2.9.2/src/SimSET.includes	2019-04-03 13:04:47.737300316 +0100
@@ -0,0 +1 @@
+.
